benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -392,8 +392,8 @@ def run(self):
         """
         try:
             self._initialize_run()
-            if self._run_type == 'test_ci':
-                self._es_index = 'bootstorm-test-ci-results'
+            if self._run_type in ('test_ci', 'chaos_ci'):
+                self._es_index = f"bootstorm-{self._run_type.replace('_', '-')}-results"
```

Comment:
Why are these cases hard coded?

Reply:
Its the dynamic elasticsearch index name
What do u mean hard coded ?

Reply:
Names of explicit run types (in this case) special cased in the code.  But why are you using `chaos-ci` rather than `chaos_ci` as the index name?  We're  not doing that with func_ci and perf_ci.

I think I remember now that this came from the elasticsearch index name.  Please add a comment

Reply:
Done.